.env.example

```diff
@@ -1,6 +1,13 @@
 # Connect to MySQL Docker container locally
 DATABASE_URL="mysql://starchart:starchart_password@localhost:3306/starchart"
 
+# Used to compute hash against the session IDs. Should be long, random.
 SESSION_SECRET="super-duper-s3cret"
 
+# https://letsencrypt.org/docs/expiration-emails/
 LETS_ENCRYPT_ACCOUNT_EMAIL="nx@senecacollege.ca"
+
+PORT=8080
+
+# One of: trace, debug, info, warn, error, fatal, silent
+LOG_LEVEL=debug
```

Comment:
Since we are not using `pino` for logging, I don't think we need this `LOG_LEVEL` environment variable.